gpustat/util.py

```diff
@@ -101,3 +102,11 @@ def report_summary(self, concise=True):
                 self._write("{msg} -> Total {value} occurrences.".format(
                     msg=msg, value=value))
             self._write('')
+
+
+def has_AMD():
+    try:
+        subprocess.check_output('rocm-smi')
```

Comment:
Is there dedicated API in rocmi to validate the environment?

Reply:
I think the answer right now is "No". We can definitely add new functions, but I am not sure what it would mean to "validate the environment"

Reply:
Talked with @bethune-bryant a bit more. Would an infalliable function (no crashing if there are missing libs/no AMDgpus) to test if there are AMD gpus on the system and return true/false work?